### ISSUE TITLE
Timezones in ubuntu based agent

### ISSUE THREAD

**Issue by hardcorehead87** [Microsoft/vsts-agent-docker#37]
We have tests running on the hosted linux preview agents that fail because timezones are missing.
When running a small test on the following docker images I get these results;

```
docker exec -it microsoft/vsts-agent:ubuntu-16.04-docker-1.12.1-standard

root@c7bad853c997:/var/temp# cat Program.cs
﻿using System;

namespace temp
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Timezone test started.");
            var timezones = TimeZoneInfo.GetSystemTimeZones();
            foreach (var timeZoneInfo in timezones)
               Console.WriteLine(timeZoneInfo.Id);
            Console.WriteLine("Timezone test done.");
        }
    }
}
root@c7bad853c997:/var/temp# dotnet run
Timezone test started.
Timezone test done.
root@c7bad853c997:/var/temp# tzselect
/usr/bin/tzselect: line 180: /usr/share/zoneinfo/iso3166.tab: No such file or directory
/usr/bin/tzselect: time zone files are not set up correctly
root@c7bad853c997:/var/temp#
```

**Comment by hardcorehead87** [comment]
As stated in the following **[issue](https://github.com/docker-library/official-images/issues/2863#issuecomment-294607003)** TZDATA has been removed from the upstream ubuntu image for space savings.
This however causes parts of dotnet to malfunction because it depends on this data being available.
The intended solution is to manually add the required package.

**Comment by hardcorehead87** [comment]
Fixed in #39

**Comment by stepro** [comment]
Merged #39.
